README.md

```diff
@@ -173,40 +173,96 @@ Create the PR, using the upstream (parent) branch from `.git/machete` as the bas
 git machete github create-pr [--draft]
 ```
 
-Synchronize with the remote repository: checkout your open PRs, delete unmanaged branches and also delete untracked managed branches with no downstream branch: <br/>
-```shell script
-git machete github sync
-```
-
 **Note**: for private repositories, a GitHub API token with `repo` access is required.
 This will be resolved from the first of:
 1. The `GITHUB_TOKEN` env var.
-2. The content of the .github-token file in the home directory (`~`). This file has to be manually created by the user.
+2. The content of the `.github-token` file in the home directory (`~`). This file has to be manually created by the user.
 3. The auth token from the current [`gh`](https://cli.github.com/) configuration.
 4. The auth token from the current [`hub`](https://github.com/github/hub) configuration.
 
 <br/>
 
+
+## FAQ
+
+#### I've run `git machete discover`... but the branch layout I see in `.git/machete` doesn't exactly match what I expected. Am I doing something wrong?
+
+[//]: # (For how to find Medium header anchors, see https://www.freecodecamp.org/news/how-to-link-to-a-specific-paragraph-in-your-medium-article-2018-table-of-contents-method-e66595fea549/)
+No! It's all right, `discover` is based on an (imperfect)
+[heuristic](https://medium.com/virtuslab/git-machete-strikes-again-traverse-the-git-rebase-jungle-even-faster-with-v2-0-f43ebaf8abb0#0544)
+which usually yields branch layout close to what the user expected...
+but it still might differ in details (like, branches discovered to be children of `main`/`develop` instead of each other).
+
+Just run [`git machete edit`](https://git-machete.readthedocs.io/en/stable/#edit) to fix the layout manually.
+If you're working on JetBrains IDEs, you can use [git-machete IntelliJ plugin](https://github.com/VirtusLab/git-machete-intellij-plugin#git-machete-intellij-plugin)
+for features like branch name completion when editing `.git/machete` file.
+
+Also, consider [`git machete github checkout-prs`](#github-integration) instead of `git machete discover` if you already have GitHub PRs opened.
+
+<br/>
+
+#### Can I use `git merge` for dealing with stacked PRs?
+
+Generally, there's a [trilemma](https://en.wikipedia.org/wiki/Trilemma) (_choose at most two out of three_):
```

Comment:
I'm a fan of fancy words, but not always in documentation for general public. Sometimes it's fine, for example:

> Rust is a systems language pursuing the trifecta: safety, concurrency, and speed.
> — https://this-week-in-rust.org/blog/2017/10/24/this-week-in-rust-205/

Even though it is imho fine, sounds cool and is rather understandable, they've ditched it for simpler:

> Rust is a programming language empowering everyone to build reliable and efficient software.
> — https://this-week-in-rust.org/blog/2022/07/20/this-week-in-rust-452/

```suggestion
Generally speaking, a Git management system can reliably provide two out of three flows:
```

Reply:
Actually i would skip 
```
Generally, there's a [trilemma](https://en.wikipedia.org/wiki/Trilemma) (choose at most two out of three):
1. stacked PRs
2. rebase/squash (generally: rewriting git history)
3. merge commits (i.e. commits [with 2+ parents](https://slides.com/plipski/git-internals#/7))
```
The 2 paragraphs below explain it very well and this is just unnecessary. (Just a small change to the second paragraph)

Reply:
(TBD as there is an alternative suggestion below)

Reply:
@PawelLipski I like the suggested change by @kgadek, it's much easier to understand.

Reply:
Actually, since the question is regarding usage of `git merge` for stacked PRs, the `2 out of 3` is unnecessary since we know we deal with stacked PR's so there are only 2 choices: rebase/squash or merge.

Reply:
```suggestion
Generally speaking, a Git management system can reliably deal with stacked PRs (e.g. PR \#1: `develop <- foo` and PR \#2: `foo <- bar`) in one of the two ways:
```

Reply:
☝🏻 lemme slightly reword that (hopefully making that even simpler)...